docs/usage_guide.rst

```diff
@@ -414,7 +418,20 @@ Additionally below the overview you get a table containing all of the taxable ev
    :align: center
 
 
-Finally you can get a nice CSV export by pressing the "Export CSV" button. This export is meant to be imported into google sheets. Press the button and then choose a directory to write the CSV files to. Once done you can import the CSV files into google sheets via its import menu.
+Finally you can get a CSV export by pressing the "Export CSV" button. This export is meant to be imported into Google Sheets. Press the button and then choose a directory to write the CSV files to. Once done you can import the CSV files into Google Sheets via its import menu. Following are definitions for the document's columns
+
+- `type` is a string describing the type of action a user engaged in. Its values can be of type `buy`, `sell` or `tx_gas_cost`.
```

Comment:
For this and all subsequent lines can you surround the type name with double backticks?

Like so: \`\`type\`\`

Also these are the current possible event types:

```python
EV_BUY = EventType('buy')
EV_SELL = EventType('sell')
EV_TX_GAS_COST = EventType('tx_gas_cost')
EV_ASSET_MOVE = EventType('asset_movement')
EV_LOAN_SETTLE = EventType('loan_settlement')
EV_INTEREST_PAYMENT = EventType('interest_rate_payment')
EV_MARGIN_CLOSE = EventType('margin_position_close')
EV_DEFI = EventType('defi_event')
```

Reply:
OK, I ended up not outlining the individual event types as that may overwhelm the reader. Let me know in case you still want them all and I'll add them.